pkg/machines/libvirt-dbus.js

```diff
@@ -1144,4 +1148,13 @@ export function storagePoolDeactivate(connectionName, objPath) {
     return call(connectionName, objPath, 'org.libvirt.StoragePool', 'Destroy', [], TIMEOUT);
 }
 
+export function storagePoolUndefine(connectionName, objPath) {
+    return call(connectionName, objPath, 'org.libvirt.StoragePool', 'Undefine', [], TIMEOUT);
+}
+
+export function storageVolumeDelete(connectionName, path) {
+    return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'StorageVolLookupByPath', [path], TIMEOUT)
+            .then(objPath => call(connectionName, objPath[0], 'org.libvirt.StorageVol', 'Delete', [0], TIMEOUT));
```

Comment:
It can fail but I have the fail handler where I call this function and I display the error in the UI.
Just for verification I replaced the method name from StorageVolLookupByPath to StorageVolLookupByPath-test and tried to delete a pool and the error I see is the following.
![vollookupbypath-fail](https://user-images.githubusercontent.com/14921356/52113875-d5ccad80-260a-11e9-8dd1-3708a045dc3f.png)